tests/Unit/Utilities/Test_StdHelpers.cpp

```diff
@@ -146,4 +147,23 @@ SPECTRE_TEST_CASE("Unit.Utilities.StdHelpers.StdArrayArithmetic",
   for (size_t i = 0; i < Dim; ++i) {
     CHECK(gsl::at(neg_p1, i) == approx(gsl::at(expected_neg_p1, i)));
   }
+
```

Comment:
Check passing different types with neither of them DataVector.

Reply:
I would go further and say we should move the DataVector tests into the file for DataVector tests, and only test a few fundamentals here.